index.bs

```diff
@@ -2411,6 +2415,17 @@ To <dfn>compute the channel capacity of a source</dfn> given a [=randomized resp
 
 Note: This algorithm computes the channel capacity [[CHAN]] of a q-ary symmetric channel [[Q-SC]].
 
+To <dfn>compute the channel capacity of a source with attribution scopes</dfn> given a positive integer |numTriggerStates|, a positive integer |attributionScopeLimit| and a positive integer |maxEventStates|:
+1. Let |totalStates| be |numTriggerStates| + |attributionScopeLimit| * |maxEventStates|.
```

Comment:
Should this be |attributionScopeLimit - 1|?

Reply:
Done.